### PR TITLE
fix: embed dashboard legacy demo

### PR DESCRIPTION
**Pull request by bryans99**
Ensure demo uses legacy dashboards.

### CODE REVIEW

**Review by jkaster — DISMISSED** [review]
Legacy FTW

File: react/javascript/demo-embeds/src/components/DashboardEmbed/DashboardEmbed.jsx [hunk 1]
```diff
@@ -112,6 +112,7 @@ export const DashboardEmbed = ({ embedType }) => {
           if (hostUrl) {
             LookerEmbedSDK.init(hostUrl)
             const embed = LookerEmbedSDK.createDashboardWithId(dashboardId)
+              .withNext('-legacy')
```
Comment (whscullin) [inline react/javascript/demo-embeds/src/components/DashboardEmbed/DashboardEmbed.jsx:115]: Maybe some of the other references, like the component name, should be updated to reflect the legacy status.

Comment (bryans99) [reply react/javascript/demo-embeds/src/components/DashboardEmbed/DashboardEmbed.jsx:118]: Makes sense. Done

**Review by whscullin — APPROVED** [review]
LGTM

**Review by jkaster — APPROVED** [review]

**Pull request merged**